test/client/client_test.go

```diff
@@ -0,0 +1,93 @@
+package client_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/openshift/cluster-logging-operator/test"
+	. "github.com/openshift/cluster-logging-operator/test/client"
+	. "github.com/openshift/cluster-logging-operator/test/matchers"
+	"github.com/openshift/cluster-logging-operator/test/object"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/watch"
+)
+
+var _ = Describe("Client", func() {
```

Comment:
Isn't `defer ginkgo.GinkgoRecover()` required for a testcase that makes use of go routines?

Reply:
It's required in the goroutine. The FailGroup type takes care of this for you so you can safely say:
```
g := test.FailGroup
g.Go(func() { Expect(blah).To(BeBoring()); }
g.Go(func() { Expect(geewhiz).To(BeGreatlyExciting()); }
g.Wait()
```
If you start goroutines by some other method then you would need the defer ginkgo stuff.